packages/design-system/src/helpers/__tests__/handleResponsiveValues.test.ts

```diff
@@ -0,0 +1,183 @@
+import { lightTheme } from '../../themes';
+import { handleResponsiveValues } from '../handleResponsiveValues';
+
+describe('handleResponsiveValues', () => {
+  it('should handle simple string values', () => {
+    const values = { padding: '1rem', margin: '4px' };
+    expect(handleResponsiveValues(values, lightTheme)).toMatchInlineSnapshot(`
+      "padding-block-start: 1rem;
+      padding-inline-end: 1rem;
+      padding-block-end: 1rem;
+      padding-inline-start: 1rem;
```

Comment:
how come we have to use `padding-X-[start|end]` can't we use `padding-X` instead? 🤔 whats the rationale behind this?

Reply:
To simplify, when we introduce array there are 4 possibilities, so I thought of just mapping to all 4 logical props, instead of just 2 by checking if there are only 2 values, otherwise map to 4.

Reply:
okay sounds good, just make sure to write all this in the mapping object so everyone else in the future knows! :)